notebooks/ml/runtime_comparison.ipynb

```diff
@@ -67,9 +65,10 @@
    "outputs": [],
    "source": [
     "results = []\n",
-    "for _ in range(30):\n",
+    "for _ in range(5):\n",
```

Comment:
was this reduction on purpose or just for testing purposes?

Reply:
Yes. Also this was the code we used to create only for one specific quantum circuit. The routine to determine the value we plan to use is generated with the cells below.